calico/datamodel_v1.py

```diff
@@ -114,6 +133,37 @@ def get_profile_id_for_profile_dir(key):
     return final_node if prefix == PROFILE_DIR else None
 
 
+def hostname_from_status_key(key):
+    """
+    Get hostname from a status key (or None if this is not a status key).
+
+    :param: key for felix status
+            expected key format: FELIX_STATUS_DIR/<hostname>/
+                                           <some path or not>/<actual key name>
+    """
+    if not key.startswith(FELIX_STATUS_DIR):
+        return False
```

Comment:
This looks like a check for an invalid status key; docstring says we return None but here we return False.